get_github_coderabbit_review/coderabbit_fetcher/processors/comment_parser.py

```
@@ -0,0 +1,347 @@
+            suggestion = self._extract_suggestion(content)
+            
+            if description:
+                return ActionableComment(
+                    id=f"actionable_{hash(content) % 10000}",
+                    description=description,
+                    suggestion=suggestion,
+                    file_path=file_info.get('path', ''),
+                    line_number=file_info.get('line', 0),
+                    raw_content=content
+                )
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)